lib/providers/services/http_service.dart

```diff
@@ -8,19 +8,29 @@ import 'package:seeds/constants/http_mock_response.dart';
 import 'package:seeds/models/models.dart';
 import 'package:seeds/providers/notifiers/voted_notifier.dart';
 
-class HttpService {
+class HttpService extends ChangeNotifier {
   String baseURL = Config.defaultEndpoint;
   String hyphaURL = Config.hyphaEndpoint;
```

Comment:
Please make a TokenNotifier and listen to this in all UX places that need to know about a change in token

Then remove token notification code from all other notifiers - so token notifier is independent of other notifiers.